### PR TITLE
[SYTO-1128] Create Unit Test for EQ Component in Mortar

### PR DESCRIPTION
**Pull request by bpearsoncse**
https://jira.natgeo.com/browse/SYTO-1128

### CODE REVIEW

File: test/src/util/EQ_test.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    before(() => {
+
+      wrapper = mount(<EQ
+        sizeClasses={sizeClasses}
+      />);
+      wrapper.setState({ contentWidth: 420 });
+    });
+
+    it('Should have a div with small class', () => {
+      expect(wrapper.find('.small-class').type()).to.equal('div');
```
Comment (pmourelle) [inline test/src/util/EQ_test.js]: Class is actually called `small-class`?

Comment (benjachapman) [inline test/src/util/EQ_test.js]: that's just one of the mock classes being used in this test I reckon - this component accepts classes as arguments

Comment (bpearsoncse) [inline test/src/util/EQ_test.js]: ^Yeah, those are just test classes.

**Comment by benjachapman** [comment]
LGTM

**Pull request merged**